### PR TITLE
OSGeo Userid password reset can be done autonomously now

### PR DESCRIPTION
**Pull request by strk**
... no need to tell people to write an email

### CODE REVIEW

**Comment by strk** [comment]
I think it is better not to add too many links (I'd actually also drop the one for changing the password) as they may change in the future. Linking to the main osgeo_userid page seems to be all is needed

File: source/site/getinvolved/development/bugreporting.rst [hunk 1]
```diff
@@ -33,8 +33,7 @@ It may require some time to have your account approved.
 .. note:: On the `OSGeo userid page <http://www.osgeo.org/osgeo_userid>`_ you can
  find more information about the OSGeo id we use. For example to 
  `change your password <https://www.osgeo.org/cgi-bin/auth/ldap_edit_user.py>`_
- or email address. Lost your password? Then request a password reset
- via info at osgeo.org
+ or email address, or to reset your lost your password.
```
Comment (DelazJ) [inline source/site/getinvolved/development/bugreporting.rst]: typo: `your lost password`

**Comment by DelazJ** [comment]
Thanks

**Comment by strk** [comment]
> typo: `your lost password`

Thanks, fixed.

**Pull request merged**